### ISSUE TITLE
Detect dead code from stub

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/javascript-analyzer#165]
The following four placeholders are present in the stubs. They should be detected and a "must-fix" message should be added to remove the line which is now dead-code.

```javascript
throw new Error('Please implement the (...) function')
throw new Error('Remove this line and implement the function')
throw new Error('Implement the (...) function')
throw new Error('Remove this statement and implement this function');
```

**Comment by CodequiryUS** [comment]
// The following placeholders indicate unimplemented or dead code and must be addressed before final submission.
// Each of these should trigger a "must-fix" message in the review process to ensure the developer removes or replaces them:
// - throw new Error('Please implement the (...) function')
// - throw new Error('Remove this line and implement the function')
// - throw new Error('Implement the (...) function')
// - throw new Error('Remove this statement and implement this function');

**Comment by IsaacG** [comment]
Is this an opportunity to switch them all to one consistent message style?

**Comment by SleeplessByte** [comment]
We would still need to support older versions, but yes, I have no issue with all of the messages being the same in the JS and TS track!

**Comment by jagdish-15** [comment]
I'll be happy to switch them all to one consistent message and raise a PR on both the JS and TS tracks for the same. Just let me know which message should be used consistently throughout the track!

**Comment by SleeplessByte** [comment]
@Cool-Katt what is your preferred stub message?

**Comment by Cool-Katt** [comment]
I aways use 
```js
throw new Error('Remove this statement and implement this function');
```
But maybe this one can also work, in the light of being more direct
```js
throw new Error('Remove this line and implement the function');
```

**Comment by SleeplessByte** [comment]
Yah I like `throw new Error('Remove this line and implement the function');`, but *technically* it's the statement to remove not the line, but I think it's fine in this case.

Let's go with

```javascript
throw new Error('Remove this line and implement the function');
```

**Comment by jagdish-15** [comment]
https://github.com/exercism/javascript/pull/2714 (JS Concept)
https://github.com/exercism/javascript/pull/2715 (JS Practice)
https://github.com/exercism/typescript/pull/1591 (TS Practice)

**Comment by jagdish-15** [comment]
All concept exercises in the TS track (there was only one, apparently) already had the correct `throw new` line, so no changes were needed there!

**Comment by naderakhgari** [comment]
Hi @SleeplessByte, I can fix this issue in PR but I think I don't have access to raise a PR. Is this still a valid issue to fix?

**Comment by SleeplessByte** [comment]
Yes, ping me in the pr and i can reopen it

**Comment by IsaacG** [comment]
You can open a PR. The bot will auto-close it. If it's been previously approved, we can reopen it.

**Comment by bpradhan-dev** [comment]
Hi @SleeplessByte! 

I was investigating this issue and noticed that **`isStubThrowStatement`** in [`src/analyzers/utils/extract_main_method.ts`](https://github.com/exercism/javascript-analyzer/blob/main/src/analyzers/utils/extract_main_method.ts) already checks for all 4 placeholder strings mentioned in the issue:

**- 'Please implement'
- 'Remove this line and implement'
- 'Implement the'
- 'Remove this statement and implement'**

The function triggers the **`REMOVE_STUB_THROW`** essential comment when these are detected. It looks like this functionality might already be implemented. Should this issue be closed?

Thanks!
